addons/mail/static/tests/tours/discuss_sub_channel_search_tour.js

```diff
@@ -6,9 +6,9 @@ import { registry } from "@web/core/registry";
 import { Deferred } from "@web/core/utils/concurrency";
 import { patch } from "@web/core/utils/patch";
 import { effect } from "@web/core/utils/reactive";
-import { contains, dragenterFiles, dropFiles, scroll } from "@web/../tests/utils";
 
 let waitForLoadMoreToDisappearDef;
+const timeout = 500;
```

Comment:
500ms isn't really much better than 200 when RPC and potential runbot slowness are involved.

There's no good reason to have a low timeout here, and all it does will be to create race conditions.

The only thing we need this timeout for is to crash relatively quickly when there are actual issues, which should be relatively infrequent (only on dev branches when related code is wrongly changed). 3s should be the minimum.

What is the current default timeout for tour step? This should likely be the same, and it should be the actual default of `waitFor` when used inside a tour step rather than having to define it manually.